internal/provider/external/installConfig.go

```diff
@@ -15,6 +16,9 @@ func (p externalProvider) AddPlatformToInstallConfig(cfg *installcfg.InstallerCo
 		None: &installcfg.PlatformNone{},
 	}
 
+	// Enable feature set "TechPreviewNoUpgrade" in order to access External platform feature
```

Comment:
@filanov This code does not exist in the master code, see: https://github.com/openshift/assisted-service/blob/master/internal/provider/external/installConfig.go

Reply:
good catch, this is why i put a hold on the PR. 
@adriengentil can you please open a manual PR for this in 2.8?

Reply:
on it

Reply:
I think we can merge this one, and cherry-pick https://github.com/openshift/assisted-service/pull/5381 afterwards. 
This code for the oci platform is outdated.

Reply:
@filanov @gamli75 what do you think?